app/sidekiq/central_mail/submit_saved_claim_job.rb

```diff
@@ -59,7 +59,7 @@ def perform(saved_claim_id)
 
     def send_claim_to_central_mail(saved_claim_id)
       @claim = SavedClaim.find(saved_claim_id)
-      @pdf_path = process_record(@claim)
+      @pdf_path = @claim.form_id == '21P-530V2' ? process_record(@claim, @claim.created_at, @claim.form_id) : process_record(@claim) # rubocop:disable Layout/LineLength
```

Comment:
I would recommend against disabling rubocop inline like this

Reply:
@LindseySaari should we wrap the method in the rubocop check?